cypress/e2e/scenario-2.1-basic-service.cy.js

```diff
@@ -234,23 +229,21 @@ describe("Scenario 2.1 Service Catalog - basic-service", () => {
     cy.get('[aria-label="Environment card"]').contains("lsm-frontend").click();
     cy.get("#basic-service").contains("Show inventory").click();
 
-    // wait to check if compile is done
-    waitForCompile();
-    cy.wait("@GetServiceInventory");
-
     // expand first row
-    cy.get("#expand-toggle0").click();
+    cy.get("#expand-toggle0", { timeout: 20000 }).click();
 
-    // There might be still a compile running because of the edit action. So we do a check if one is pending.
     waitForCompile();
```

Comment:
why this one stays, and other is deleted?

Reply:
cause I'm sure 100% that one would be running at least. So it's a bit of a check to see if a compile is triggered if that makes sense.

Reply:
Makes sense :D